.bazelrc

```diff
@@ -1,2 +1,2 @@
-build --java_toolchain=@bazel_tools//tools/jdk:toolchain_java8
-build --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_java8
\ No newline at end of file
+build --java_toolchain=@bazel_tools//tools/jdk:toolchain_java9
```

Comment:
why not 11?

Reply:
https://www.baeldung.com/java-9-process-api#triggering-dependent-actions-on-process-termination

Reply:
(we don't use it yet, but we discussed with @Abrams27 that we'd like to change some async logic in the server and this would allow for some improvements, but of course this method is not indispensable)

Reply:
Ah, I was not aware Java9 had introduced such an API, I agree with its use, not sure if we can abdicate the use of java8 just like that, I'd advise to discuss this with @jastice

Reply:
if only we could update java version then we should use 11 (LTS) instead of 9